Nile last version is not compatible with starknet cairo last version for deploying Contracts

Was deploying a custom ERC20 token : 

`nile deploy DAOToken --network goerli 4918299492130448750 1145130836 0 100000000 0`
🚀 Deploying DAOToken

Also Tried : 
`nile deploy DAOToken --no_wallet --network goerli 4918299492130448750 1145130836 0 100000000 0`
`nile deploy DAOToken --wallet XXX --network goerli 4918299492130448750 1145130836 0 100000000 0`



Error : 

```
Error: AssertionError: A wallet must be specified (using --wallet or the STARKNET_WALLET environment variable), unless specifically using --no_wallet.
Traceback (most recent call last):
...

subprocess.CalledProcessError: Command '['starknet', 'deploy', '--contract', 'artifacts/DAOToken.json', '--inputs', '4918299492130448750', '1145130836', '0', '100000000', '0']' returned non-zero exit status 1.
```



---
Nile version : 0.6.1
starknet version : 0.9.1

--- a/src/nile/common.py
+++ b/src/nile/common.py
@@ -67,6 +67,8 @@ def run_command(
     else:
         command.append(f"--gateway_url={GATEWAYS.get(network)}")
 
+    command.append("--no_wallet")
+
     return subprocess.check_output(command)
 
 

--- a/tests/test_common.py
+++ b/tests/test_common.py
@@ -1,45 +1,40 @@
 """Tests for deploy command."""
 import logging
 from unittest.mock import patch
-import os
+
 import pytest
 
-from nile.common import run_command, BUILD_DIRECTORY
-
+from nile.common import BUILD_DIRECTORY, run_command
 
 CONTRACT = "contract"
 OPERATION = "invoke"
 NETWORK = "goerli"
 ARGS = [1, 2, 3]
 
-@pytest.fixture(autouse=True)
-def tmp_working_dir(monkeypatch, tmp_path):
-    monkeypatch.chdir(tmp_path)
-    return tmp_path
+
+#@pytest.fixture(autouse=True)
+#def tmp_working_dir(monkeypatch, tmp_path):
+#    monkeypatch.chdir(tmp_path)
+#    return tmp_path
 
 
-@pytest.mark.parametrize(
-    "operation", ["invoke", "call"]
-)
+@pytest.mark.parametrize("operation", ["invoke", "call"])
 @patch("nile.common.subprocess.check_output")
 def test_run_command(mock_subprocess, operation):
     logging.getLogger().setLevel(logging.INFO)
 
     run_command(
-        contract_name=CONTRACT,
-        network=NETWORK,
-        operation=operation,
-        arguments=ARGS
+        contract_name=CONTRACT, network=NETWORK, operation=operation, arguments=ARGS
     )
 
-    mock_subprocess.assert_called_once_with([
-        "starknet",
-        operation,
-        "--contract",
-        f"{BUILD_DIRECTORY}/{CONTRACT}.json",
-        "--inputs",
-        *ARGS,
-        "--no_wallet"
-    ])
-
-
+    mock_subprocess.assert_called_once_with(
+        [
+            "starknet",
+            operation,
+            "--contract",
+            f"{BUILD_DIRECTORY}/{CONTRACT}.json",
+            "--inputs",
+            *ARGS,
+            "--no_wallet",
+        ]
+    )

--- a/tests/test_common.py
+++ b/tests/test_common.py
@@ -1,5 +1,4 @@
 """Tests for deploy command."""
-import logging
 from unittest.mock import patch
 
 import pytest
@@ -12,16 +11,9 @@ NETWORK = "goerli"
 ARGS = [1, 2, 3]
 
 
-#@pytest.fixture(autouse=True)
-#def tmp_working_dir(monkeypatch, tmp_path):
-#    monkeypatch.chdir(tmp_path)
-#    return tmp_path
-
-
 @pytest.mark.parametrize("operation", ["invoke", "call"])
 @patch("nile.common.subprocess.check_output")
 def test_run_command(mock_subprocess, operation):
-    logging.getLogger().setLevel(logging.INFO)
 
     run_command(
         contract_name=CONTRACT, network=NETWORK, operation=operation, arguments=ARGS

--- a/tests/test_common.py
+++ b/tests/test_common.py
@@ -1,0 +1,45 @@
+"""Tests for deploy command."""
+import logging
+from unittest.mock import patch
+import os
+import pytest
+
+from nile.common import run_command, BUILD_DIRECTORY
+
+
+CONTRACT = "contract"
+OPERATION = "invoke"
+NETWORK = "goerli"
+ARGS = [1, 2, 3]
+
+@pytest.fixture(autouse=True)
+def tmp_working_dir(monkeypatch, tmp_path):
+    monkeypatch.chdir(tmp_path)
+    return tmp_path
+
+
+@pytest.mark.parametrize(
+    "operation", ["invoke", "call"]
+)
+@patch("nile.common.subprocess.check_output")
+def test_run_command(mock_subprocess, operation):
+    logging.getLogger().setLevel(logging.INFO)
+
+    run_command(
+        contract_name=CONTRACT,
+        network=NETWORK,
+        operation=operation,
+        arguments=ARGS
+    )
+
+    mock_subprocess.assert_called_once_with([
+        "starknet",
+        operation,
+        "--contract",
+        f"{BUILD_DIRECTORY}/{CONTRACT}.json",
+        "--inputs",
+        *ARGS,
+        "--no_wallet"
+    ])
+
+

--- a/tox.ini
+++ b/tox.ini
@@ -13,7 +13,7 @@ passenv =
 extras =
     testing
 deps =
-    cairo-lang==0.9.0
+    cairo-lang==0.9.1
     starknet-devnet
     # See https://github.com/starkware-libs/cairo-lang/issues/52
     marshmallow-dataclass==8.5.3
@@ -71,5 +71,5 @@ deps =
     black
     isort
 commands =
-    isort src/nile tests 
-    black src/nile tests 
+    isort src/nile tests
+    black src/nile tests

